manual/src/ornate/reference.md

```diff
@@ -64,24 +64,97 @@ useYarn := true
 
 Yarn 0.22.0+ must be available on the host platform.
 
-### Reload Workflow {#reload-workflow}
+### Bundling Modes
+
+Each time you change something in your application source code and compile the project, Scala.js emits a new .js 
+file that can weigh several MBs if your application is large. Scalajs-bundler provides
+a few different options with respect to handling this large output file.
+
+#### Application (default) {#bundling-mode-application}
+
+`webpackBundlingMode := BundlingMode.Application` generates a webpack config that simply processes the scala.js output file as an 
+entrypoint. This means that webpack loaders, plugins, etc will work as usual. It also means that webpack will 
+have to process a very large Scala.Js output file.
+
+Turning this CommonJS module into code executable by web browsers takes time, often upwards of a minute. 
+
+Nonetheless, this is identical to what you'd get if were to duplicate your workflow outside scalajs-bundler, 
+so it remains the default. 
+
+#### Library Only {#bundling-mode-library-only}
+You can get a much faster “change source and reload application” workflow by setting the 
+`webpackBundlingMode := BundlingMode.LibraryOnly()` key. This bundling mode avoids having webpack process the entire
+scala.js output, but instead uses webpack to bundle all the javascript dependencies (determined via `@JSImport` 
+and any changes to the `webpack.config.js`). This is accomplished by setting the webpack `output.library` and 
+`output.libraryTarget` keys in the webpack.config. By default, this generates a global variable named
+`ScalaJSBundlerLibrary`, but if needed, that variable name can be overridden via the `exportedName` 
+parameter provided to `BundlingMode.LibraryOnly`.
+
+It then uses a special "loader" to resolve these dependencies for the actual scala.js application.
+
+In order to use this mode, instead of including `yourapp-bundle.js` in your page, you will need to include
+`yourapp-library.js`, `yourapp-loader.js`, `yourapp-fastopt.js` or `yourapp-opt.js`. If you're using Play 
+Framework, you could use a twirl template similar to this one:
+
+~~~ html
+@(projectName: String,
+  assets: String => String,
+  resourceExists: String => Boolean,
+  htmlAttributes: Html = Html("")
+)
+
+@defining(s"${projectName.toLowerCase}") { name =>
+  @Seq(s"$name-opt-library.js", s"$name-fastopt-library.js").find(resourceExists).map(name => jsScript(assets(name), htmlAttributes))
+}
+<script language="JavaScript">
+var exports = window;
+exports.require = window["ScalaJsBundlerDependencies"].require;
```

Comment:
Isn't this `ScalaJSBundlerLibrary`?

Reply:
Argh, yep. Wish there was a better way to keep such things in sync. Fix pushed.